focus-android/app/src/main/java/org/mozilla/focus/locale/screen/LocaleDescriptor.kt

```diff
@@ -28,34 +28,34 @@ class LocaleDescriptor(private val localeTag: String) : Comparable<LocaleDescrip
         languageCodeAndNameMap["ast"] = "Asturianu"
         // On an Android 8.0 device those languages are not known and we need to add the names
         // manually. Loading the resources at runtime works without problems though.
+        languageCodeAndNameMap["ace"] = "Acehnese"
+        languageCodeAndNameMap["an"] = "Aragonés"
+        languageCodeAndNameMap["anp"] = "अंगिका"
+        languageCodeAndNameMap["ay"] = "Aimara"
         languageCodeAndNameMap["cak"] = "Kaqchikel"
+        languageCodeAndNameMap["co"] = "Corsu"
+        languageCodeAndNameMap["hus"] = "Tének"
         languageCodeAndNameMap["ia"] = "Interlingua"
+        languageCodeAndNameMap["ixl"] = "Ixil"
+        languageCodeAndNameMap["jv"] = "Basa Jawa"
         languageCodeAndNameMap["meh"] = "Tu´un savi ñuu Yasi'í Yuku Iti"
         languageCodeAndNameMap["mix"] = "Tu'un savi"
-        languageCodeAndNameMap["trs"] = "Triqui"
-        languageCodeAndNameMap["zam"] = "DíɁztè"
+        languageCodeAndNameMap["nv"] = "Navajo"
         languageCodeAndNameMap["oc"] = "occitan"
-        languageCodeAndNameMap["an"] = "Aragonés"
-        languageCodeAndNameMap["tt"] = "татарча"
-        languageCodeAndNameMap["wo"] = "Wolof"
-        languageCodeAndNameMap["anp"] = "अंगिका"
-        languageCodeAndNameMap["ixl"] = "Ixil"
         languageCodeAndNameMap["pai"] = "Paa ipai"
-        languageCodeAndNameMap["quy"] = "Chanka Qhichwa"
-        languageCodeAndNameMap["ay"] = "Aimara"
-        languageCodeAndNameMap["quc"] = "K'iche'"
-        languageCodeAndNameMap["tsz"] = "P'urhepecha"
-        languageCodeAndNameMap["jv"] = "Basa Jawa"
         languageCodeAndNameMap["ppl"] = "Náhuat Pipil"
-        languageCodeAndNameMap["su"] = "Basa Sunda"
-        languageCodeAndNameMap["hus"] = "Tének"
-        languageCodeAndNameMap["co"] = "Corsu"
+        languageCodeAndNameMap["quc"] = "K'iche'"
+        languageCodeAndNameMap["quy"] = "Chanka Qhichwa"
+        languageCodeAndNameMap["skr"] = "سرائیکی"
         languageCodeAndNameMap["sn"] = "ChiShona"
+        languageCodeAndNameMap["su"] = "Basa Sunda"
+        languageCodeAndNameMap["trs"] = "Triqui"
+        languageCodeAndNameMap["tsz"] = "P'urhepecha"
+        languageCodeAndNameMap["tt"] = "татарча"
+        languageCodeAndNameMap["wo"] = "Wolof"
+        languageCodeAndNameMap["zam"] = "DíɁztè"
         languageCodeAndNameMap["zh-CN"] = "中文 (中国大陆)"
-        languageCodeAndNameMap["nv"] = "Navajo"
-        languageCodeAndNameMap["skr"] = "سرائیکی"
     }
-
```

Comment:
We probably don't want to remove this new line that separates the function

Reply:
👍🏽